keps/sig-scheduling/3633-matchlabelkeys-to-podaffinity/README.md

```diff
@@ -758,13 +758,24 @@ rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
 
+It shouldn't impact already running workloads. It's an opt-in feature,
+and users need to set `matchLabelKeys` or `mismatchLabelKeys` field in PodAffinity or PodAntiAffinity to use this feature. 
```

Comment:
Furthermore, the node selectors are not modified back by the apiserver for already created Pods.